### PR TITLE
[ja] Probabilities and Statistics

### PR DESCRIPTION
**Pull request by nao0811ta**
I've finished translating. please, someone, review this.

### CODE REVIEW

**Comment by shervinea** [comment]
Great work @nao0811ta, thank you for all your time! Let's start the review process now!

**Comment by nao0811ta** [comment]
Gogo. Could you please set review label? I don’t have permission to set it.

**Comment by shervinea** [comment]
@nao0811ta Done! Please feel free to invite anyone you know who may review your work.

**Comment by suzana-ilic** [comment]
Hi @shervinea all of the cheatsheets will be translated into Japanese by MLT members - if you set the label to "reviewer wanted" for the rest as well I can send out an open call! Thanks so much!

**Comment by shervinea** [comment]
@suzana-ilic that's awesome, thank you so much for your help! As of now, we are looking for reviewers for:
- #145: Convolutional neural networks
- #146: Recurrent neural networks
- #157: Deep learning tips and tricks
- #96: Deep learning
- #142: Probabilities and Statistics
- #140: Linear Algebra

current work in progress are:
- #144: Supervised learning
- #99: Machine learning tips and tricks

and the translation yet to be started is:
- Unsupervised learning

**Comment by ytknzw** [comment]
Hi @shervinea and @suzana-ilic 
I review #142: Probabilities and Statistics.

File: ja/refresher-probability.md [hunk 1]
```diff
@@ -0,0 +1,381 @@
...
+
+**38. [Case, Marginal density, Cumulative function]**
+
+&#10230;[例,、周辺密度、累積関数]
+
+<br>
+
+**39. Conditional density ― The conditional density of X with respect to Y, often noted fX|Y, is defined as follows:**
+
+&#10230;条件部密度(Conditional density) - Yに対するXの条件部密度は主にfx|Yと表記され、次のように定義されます:
```
Comment (ytknzw) [inline ja/refresher-probability.md]: "条件部密度" -> "条件付き密度"
"主に" -> "よく"
"fx|Y" -> "fX|Y"

File: ja/refresher-probability.md [hunk 2]
```diff
@@ -0,0 +1,381 @@
...
+
+**41. Covariance ― We define the covariance of two random variables X and Y, that we note σ2XY or more commonly Cov(X,Y), as follows:**
+
+&#10230;共分散(Covariance) - 次のようにふたつの確率変数X,Yの共分散をσ2XYまたはさらに一般的にはCov(X,Y)で定義します。
+
+<br>
+
+**42. Correlation ― By noting σX,σY the standard deviations of X and Y, we define the correlation between the random variables X and Y, noted ρXY, as follows:**
+
+&#10230;相関関係(Correlation) - X, Yの標準変数をσX,σYで表記し、確率変数X,Yの相関関係をρXYで表記し、次のように定義します。
```
Comment (ytknzw) [inline ja/refresher-probability.md]: "相関関係" -> "相関係数"
"標準変数" -> "標準偏差"
"で表記し" -> "と表記し"
"定義します。" -> "定義します:"

File: ja/refresher-probability.md [hunk 3]
```diff
@@ -0,0 +1,381 @@
...
+
+<br>
+
+**43. Remark 1: we note that for any random variables X,Y, we have ρXY∈[−1,1].**
+
+&#10230;注釈 1: 任意の確率変数X,Yに対してρXY∈[−1,1]となります。
```
Comment (ytknzw) [inline ja/refresher-probability.md]: "となります" -> "が成り立ちます"

File: ja/refresher-probability.md [hunk 4]
```diff
@@ -0,0 +1,381 @@
...
+
+<br>
+
+**44. Remark 2: If X and Y are independent, then ρXY=0.**
+
+&#10230;注釈 2: XとYが独立ならρXY=0です。
```
Comment (ytknzw) [inline ja/refresher-probability.md]: "なら" -> "ならば、"

File: ja/refresher-probability.md [hunk 5]
```diff
@@ -0,0 +1,381 @@
...
+
+**46. Definitions**
+
+&#10230;定義
+
+<br>
+
+**47. Random sample ― A random sample is a collection of n random variables X1,...,Xn that are independent and identically distributed with X.**
+
+&#10230;確率標本(Random sample) - 確率標本はXと独立で同一に分布するn個の確率変数X1,...,Xnの集まりです。
```
Comment (ytknzw) [inline ja/refresher-probability.md]: "確率標本は" -> "確率標本とは"
"Xと独立で同一に分布する" -> "Xに従う独立同分布の"
"集まり" -> "集合"

File: ja/refresher-probability.md [hunk 6]
```diff
@@ -0,0 +1,381 @@
...
+
+<br>
+
+**48. Estimator ― An estimator is a function of the data that is used to infer the value of an unknown parameter in a statistical model.**
+
+&#10230;推定量(Estimator) - 推定量は統計モデルで未知のパラメータの値を推定するために使用されるデータの関数です。
```
Comment (ytknzw) [inline ja/refresher-probability.md]: "推定量は" -> "推定量とは"
"統計モデルで" -> "統計モデルにおける"
"推定するために使用される" -> "推定するのに用いられる" for consistency.

File: ja/refresher-probability.md [hunk 7]
```diff
@@ -0,0 +1,381 @@
...
+
+<br>
+
+**49. Bias ― The bias of an estimator ^θ is defined as being the difference between the expected value of the distribution of ^θ and the true value, i.e.:**
+
+&#10230;偏り(Bias) - 推定量^θの偏りは^θの期待値と実際の値との差で定義されます。
```
Comment (ytknzw) [inline ja/refresher-probability.md]: "^θの期待値" -> "^θのの分布の期待値"
"実際の値" -> "真の値"
"で定義されます。" -> "として定義されます。すなわち:"

File: ja/refresher-probability.md [hunk 8]
```diff
@@ -0,0 +1,381 @@
...
+
+<br>
+
+**50. Remark: an estimator is said to be unbiased when we have E[^θ]=θ.**
+
+&#10230;注釈: 推定量はE[^θ]=θの場合、不偏といいます。
```
Comment (ytknzw) [inline ja/refresher-probability.md]: "推定量はE[^θ]=θの場合、不偏といいます。" -> "E[^θ]=θが成り立つとき、推定量は不偏であるといいます。"

File: ja/refresher-probability.md [hunk 9]
```diff
@@ -0,0 +1,381 @@
...
+
+**51. Estimating the mean**
+
+&#10230;平均の推定
+
+<br>
+
+**52. Sample mean ― The sample mean of a random sample is used to estimate the true mean μ of a distribution, is often noted ¯¯¯¯¯X and is defined as follows:**
+
+&#10230;標本平均(Sample mean) - 確率標本の標本平均は実の平均μを推定するのに用いられ、主に¯¯¯¯¯Xと表記され次のように定義されます。
```
Comment (ytknzw) [inline ja/refresher-probability.md]: "は実の平均" -> "は、ある分布の真の平均"
"主に" -> "よく"
"表記され" -> "表記され、"
"定義されます。" -> "定義されます:"

File: ja/refresher-probability.md [hunk 10]
```diff
@@ -0,0 +1,381 @@
...
+
+<br>
+
+**53. Remark: the sample mean is unbiased, i.e E[¯¯¯¯¯X]=μ.**
+
+&#10230;注釈: 標本平均は不偏です。すなわちE[¯¯¯¯¯X]=μとなります。
```
Comment (ytknzw) [inline ja/refresher-probability.md]: "となります。" -> "が成り立ちます。"

File: ja/refresher-probability.md [hunk 11]
```diff
@@ -0,0 +1,381 @@
...
+
+<br>
+
+**54. Central Limit Theorem ― Let us have a random sample X1,...,Xn following a given distribution with mean μ and variance σ2, then we have:**
+
+&#10230;中心極限定理 - 平均μと分散σ2を持つ分布を従う確率標本X1,...,Xnがある。その場合、次のようになります。
```
Comment (ytknzw) [inline ja/refresher-probability.md]: "平均μと分散σ2を持つ分布を従う確率標本X1,...,Xnがある。" -> "確率標本X1,...,Xnが平均μと分散σ2を持つある分布に従うとすると、"
"その場合、次のようになります。" -> "次が成り立ちます:"

File: ja/refresher-probability.md [hunk 12]
```diff
@@ -0,0 +1,381 @@
...
+
+<br>
+
+**55. Estimating the variance**
+
+&#10230;分散推定
```
Comment (ytknzw) [inline ja/refresher-probability.md]: "分散推定" -> "分散の推定"

File: ja/refresher-probability.md [hunk 13]
```diff
@@ -0,0 +1,381 @@
...
+
+<br>
+
+**56. Sample variance ― The sample variance of a random sample is used to estimate the true variance σ2 of a distribution, is often noted s2 or ^σ2 and is defined as follows:**
+
+&#10230;標本分散 - 確率標本の標本分散は実の分散σ2を推定するのに用いられ、主にs2または^σ2と表記し次のように定義されます。
```
Comment (ytknzw) [inline ja/refresher-probability.md]: "は実の分散" -> "は、ある分布の真の分散"
"主に" -> "よく"
"表記し" -> "表記され、" for consistency.
"定義されます。" -> "定義されます:"

File: ja/refresher-probability.md [hunk 14]
```diff
@@ -0,0 +1,381 @@
...
+
+<br>
+
+**57. Remark: the sample variance is unbiased, i.e E[s2]=σ2.**
+
+&#10230;注釈: 標本分散は不偏です。つまりE[s2]=σ2になります。
```
Comment (ytknzw) [inline ja/refresher-probability.md]: "つまり" -> "すなわち" for consistency.
"になります。" -> "が成り立ちます。"

File: ja/refresher-probability.md [hunk 15]
```diff
@@ -0,0 +1,381 @@
...
+
+<br>
+
+**58. Chi-Squared relation with sample variance ― Let s2 be the sample variance of a random sample. We have:**
+
+&#10230;標本分散とカイ二乗の関係 - 確率標本の標本分散をs2としよう。次のようになります。
```
Comment (ytknzw) [inline ja/refresher-probability.md]: "カイ二乗の関係" -> "カイ二乗分布との関係"
"としよう。次のようになります。" -> "とすると、次が成り立ちます:"

File: ja/refresher-probability.md [hunk 16]
```diff
@@ -0,0 +1,381 @@
...
+
+<br>
+
+**59. [Introduction, Sample space, Event, Permutation]**
+
+&#10230;[紹介、標本空間、事象、順列]
```
Comment (ytknzw) [inline ja/refresher-probability.md]: "紹介" -> "導入"

File: ja/refresher-probability.md [hunk 17]
```diff
@@ -0,0 +1,381 @@
...
+
+<br>
+
+**60. [Conditional probability, Bayes' rule, Independence]**
+
+&#10230;[条件部確率、ベイズの定理、独立]
```
Comment (ytknzw) [inline ja/refresher-probability.md]: "条件部確率" -> "条件付き確率"

File: ja/refresher-probability.md [hunk 18]
```diff
@@ -0,0 +1,381 @@
...
+
+**62. [Probability distributions, Chebyshev's inequality, Main distributions]**
+
+&#10230;[確率分布、チェビシェフの不等式、主な分布]
+
+<br>
+
+**63. [Jointly distributed random variables, Density, Covariance, Correlation]**
+
+&#10230;[結合分布の確率変数、密度、共分散、相関関係]
```
Comment (ytknzw) [inline ja/refresher-probability.md]: "結合分布の確率変数" -> "同時分布の確率変数" for consistency.
"相関関係" -> "相関係数"

**Comment by ytknzw** [comment]
Hi @shervinea and @nao0811ta
Review for #142: Probabilities and Statistics is finished.

**Comment by shervinea** [comment]
Thank you @ytknzw and @nao0811ta for your thorough work!
@nao0811ta please feel free to let me know when you think the PR is ready to be merged.

**Comment by suzana-ilic** [comment]
Hi @nao0811ta! Just a quick reminder, please take a look a the comments and let us know when it's ready to be merged! Thanks so much! :)

**Comment by nao0811ta** [comment]
@ytknzw san
thank you for your work sincerely.

**Comment by shervinea** [comment]
Thank you both @nao0811ta and @ytknzw for your amazing work. Merging the PR right now.

**Pull request merged**